vscode-extension/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "aiconfig-editor",
```

Comment:
nit, this is pretty similar to our existing @lastmileai/aiconfig-editor package name. Should this be aiconfig-editor-vscode or something to distinguish this as the vscode extension?

Reply:
Ah great callout, thanks! Will rename to `vscode-aiconfig`